index.js

```diff
@@ -162,23 +163,40 @@ const writeIndexPage = (argv, input) => {
  * @return {boolean} => returns true if the user input is valid (valid input and output directory) 
  */
 const getUserInput = (argv) => {
-    let input = argv.input.join(' ');
+    let input = ''; 
     let filesArray = [];
 
-    // Setting the output directory
-    if (argv.output) {
-        if (!fs.existsSync(argv.output)) {
+    if (argv.input) {
+        input = argv.input.join(' ');
+        // Setting the output directory
+        if (argv.output && !fs.existsSync(argv.output)) {
+            console.error('Invalid output argument entered.');
             return false;
+        } else {
+            argv.output = 'dist';
+        }
+        // Setting the stylesheet
+        if (!argv.stylesheet) {
+            argv.stylesheet = 'style.css';
         }
-    } else {
-        argv.output = 'dist';
     }
-    // Setting the stylesheet
-    if (!argv.stylesheet) {
-        argv.stylesheet = 'style.css';
+
+    if(argv.config && fs.existsSync(argv.config)) {
+        if (fs.statSync(argv.config).isFile() && path.extname(input) == '.json') {
```

Comment:
Hey @menghif, I just noticed one error in this line. It should be path.extname(argv.config) since input is an empty string.

Reply:
Fixed! Sorry about that